src/renderer/hooks/usePaneNodeSearchMenu.tsx

```diff
@@ -0,0 +1,325 @@
+                            display: searchQuery ? undefined : 'none',
+                            fontSize: '66%',
+                        }}
+                        onClick={() => setSearchQuery('')}
+                    >
+                        <CloseIcon />
+                    </InputRightElement>
+                </InputGroup>
+                <Box
+                    h="auto"
+                    maxH={400}
+                    overflowY="scroll"
+                    p={1}
+                >
+                    {[...byCategories].length > 0 ? (
+                        [...byCategories].map(([category, categoryNodes]) => {
+                            const accentColor = getNodeAccentColors(category);
+                            return (
+                                <Box key={category}>
+                                    <HStack
+                                        borderRadius="md"
+                                        mx={1}
+                                        py={0.5}
+                                    >
+                                        <IconFactory
+                                            accentColor={accentColor}
+                                            boxSize={3}
+                                            icon={category}
+                                        />
+                                        <Text fontSize="xs">{category}</Text>
+                                    </HStack>
+                                    {[...categoryNodes].map((node) => (
+                                        <HStack
+                                            _hover={{
+                                                backgroundColor: bgColor,
+                                            }}
+                                            borderRadius="md"
+                                            key={node.schemaId}
+                                            mx={1}
+                                            px={2}
+                                            py={0.5}
+                                            onClick={() => {
+                                                setSearchQuery('');
+                                                onPaneContextMenuNodeClick(node, mousePosition);
+                                            }}
+                                        >
+                                            <IconFactory
+                                                accentColor="gray.500"
+                                                icon={node.icon}
+                                            />
+                                            <Text>{node.name}</Text>
+                                        </HStack>
+                                    ))}
+                                </Box>
+                            );
+                        })
+                    ) : (
+                        <Center w="full">No compatible nodes found.</Center>
```

Comment:
Could you make the text less bright? I feel like 

![image](https://user-images.githubusercontent.com/20878432/174854394-5a79aac6-e9fd-4cef-a5ca-292afad2637b.png)

is a bit too bright for what is just a small note.